src/ui/components/address-action/ApplicationLine/ApplicationLine.tsx

```diff
@@ -1,22 +1,23 @@
-import React, { useLayoutEffect, useState } from 'react';
 import { animated, useTransition } from '@react-spring/web';
 import { capitalize } from 'capitalize-ts';
+import { ethers } from 'ethers';
```

Comment:
The only relevant change here is the `import { ethers } from 'ethers';`
Others are because I used `organize imports` code action (btw it really organizes imports :))